lib/client/hostendpoint.go

```diff
@@ -0,0 +1,53 @@
+package client
+
+import (
+	. "github.com/projectcalico/libcalico/lib/api"
+)
+
+// HostEndpointInterface has methods to work with HostEndpoint resources.
+type HostEndpointInterface interface {
```

Comment:
I don't think you should put "Interface" on the end of an interface name.  What is this, TheJavaProgrammingLanguage?